salesforce/modules/rest/stream_implementer.bal

```diff
@@ -149,3 +149,4 @@ type SoslRecordData record {|
 |};
 
 
+
```

Comment:
Remove this extra line

Reply:
Fixed in 8bfb122ccf270a0868c40a7ea6ddaef1d3a79a39